DataAccessLayer/classes/MailChimpManager.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using Mandrill;
+using Microsoft.Extensions.Configuration;
+using Models;
+
+namespace DataAccessLayer
+{
+    public class MailChimpManager : IMailManager
```

Comment:
why do we have a mailchimp file?

Reply:
Legacy code: File deleted and Dependencies removed